src/annotator/guest.coffee

```diff
@@ -115,6 +126,54 @@ module.exports = class Guest extends Delegator
       if not @plugins[name] and @options.pluginClasses[name]
         this.addPlugin(name, opts)
 
+    # Setup event handlers on the root element
+    this._elementEventListeners = []
+    this._setupElementEvents()
+
+
+  # Add DOM event listeners for clicks, taps etc. on the document and
+  # highlights.
+  _setupElementEvents: ->
+    addListener = (event, callback) =>
+      @element[0].addEventListener(event, callback)
+      this._elementEventListeners.push({ event, callback })
+
+    # Hide the sidebar in response to a document click or tap, so it doesn't obscure
+    # the document content.
+    maybeHideSidebar = (event) =>
+      if !this.closeSidebarOnDocumentClick || this.isEventInAnnotator(event) || @selectedTargets?.length
+        # Don't hide the sidebar if event occurred inside Hypothesis UI, or
+        # the user is making a selection, or the behavior was disabled because
+        # the sidebar doesn't overlap the content.
+        return
+      @crossframe?.call('hideSidebar')
+
+    addListener 'click', (event) =>
+      annotations = annotationsAt(event.target)
+      if annotations.length and @visibleHighlights
+        xor = event.metaKey or event.ctrlKey
```

Comment:
The naming of this variable gives me a pedantic itch. These things are mutually exclusive at a very high level, but this is an odd application of the term `xor`. Perhaps `modifierKey` or something? Yeah, I know you inherited this!

Amended: Zoinks! I had no idea this metakey behavior existed! TIL!

Reply:
I believe `xor` was a cute shorthand for "toggle the current selected state". I'll rename this.